q2_annotate/busco/busco.py

```diff
@@ -140,6 +154,24 @@ def _evaluate_busco(
     common_args = _process_common_input_params(
         processing_func=_parse_busco_params, params=kwargs
     )
+    if isinstance(mags, MultiMAGSequencesDirFmt):
+        busco_results = _busco_helper(mags, common_args, additional_metrics)
+        busco_results["unbinned_contigs"] = pd.NA
```

Comment:
The entire content of this if branch should probably just become a small, independent function. Please refactor this and add a unit test for this specific functionality. This will make the code more modular and easy to follow.